client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -327,7 +327,7 @@ function HostingConfigurationSubmenu( { site, recordTracks }: SitesMenuItemProps
 					product_slug: site.plan?.product_slug,
 				} }
 			/>
-			<MenuItemLink info={ displayUpsell && __( 'Requires a Business Plan' ) }>
+			<MenuItemLink info={ displayUpsell ? __( 'Requires a Business Plan' ) : undefined }>
```

Comment:
The `info` prop accepts either a string or `undefined`, so a boolean was not a valid value.